graphics/src/test/kotlin/ktx/graphics/graphicsTest.kt

```diff
@@ -65,6 +67,21 @@ class GraphicsTest {
     val batch = mock<Batch>()
 
     batch.use {
+      verify(batch, never()).projectionMatrix = null // should not be set if none passed
```

Comment:
Use `any` matcher instead of null, as we want to make sure that the matrix is never changed.